Assets/ConnectApp/Screens/PersonalScreen.cs

```diff
@@ -216,24 +213,21 @@ public class _PersonalScreenState : AutomaticKeepAliveClientMixin<PersonalScreen
             );
         }
 
-        Widget _buildQrScanWidget(EdgeInsets padding) {
+        Widget _buildQrScanWidget() {
             if (!this.widget.viewModel.scanEnabled) {
-                return new Container();
+                return new Container(height: 60);
             }
 
             return new Row(
                 mainAxisAlignment: MainAxisAlignment.end,
                 children: new List<Widget> {
-                    new GestureDetector(
-                        onTap: QRScanPlugin.PushToQRScan,
-                        child: new Container(
-                            padding: padding,
-                            color: CColors.Transparent,
-                            child: Image.asset(
-                                "image/scan-qr-code",
-                                width: 20,
-                                height: 20
-                            )
+                    new CustomButton(
+                        padding: EdgeInsets.only(16, 16, 20, 16),
+                        onPressed: QRScanPlugin.PushToQRScan,
```

Comment:
any potential problem if user press the button multiple times in a short period of time?

Reply:
for all call API actions, should we add a cold-down like mechanism to the action triggers?

Reply:
> any potential problem if user press the button multiple times in a short period of time?

I try it, no bad things happened. but your advice makes sense.

Reply:
ok, then let's just leave it :-)

Reply:
> for all call API actions, should we add a cold-down like mechanism to the action triggers?

I'm not sure if there is, but maby we need that.

Reply:
@zhuxingwei I will record that good advice.